### PR TITLE
Sql sequelize

### PR DESCRIPTION
**Pull request by 235002**

### CODE REVIEW

File: server/routes/testSequelizer.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const { Project, User, Team } = require("../services/dbConnection");
+
+const user = {name: "Jakub", surname: "Pawłowski", team_id: 2};
```
Comment (arturfracala) [inline server/routes/testSequelizer.js]: Shouldn't we put test files in separate directories?

Comment (235002) [reply server/routes/testSequelizer.js]: This file shouldn't be merged.

Comment (235002) [reply server/routes/testSequelizer.js]: It has been deleted from branch.

File: server/routes/project.js [hunk 2]
```diff
@@ -44,20 +40,24 @@ const projectRoutes = app => {
       tags,
       theme_color
     } = req.body.project;
-    const database = new DBConnection();
-    database
-      .query("SELECT id FROM project ORDER BY id DESC LIMIT 1")
+    console.log(req.body.project);
```
[the remark this answers is not in the document]
Comment (235002) [reply server/routes/project.js]: You are correct, it shoudn't be here.

File: server/routes/github.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ const DBConnection = require("../services/DBConnection");
 const { getCode, getState, getToken } = require("../utils/selectors");
 const crypto = require("crypto");
 
+const User = require("../services/dbConnection");
```
[the remark this answers is not in the document]
Comment (235002) [reply server/routes/github.js]: Yeah, you right, i changed some queries to sequelize

File: server/routes/project.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (235002) [reply server/routes/project.js]: Like in github.js, that query has been rewirte to sequelize

**Comment by Imperative2** [comment]
Doesn't work on my side

**Pull request merged**